### ISSUE TITLE
data preprocessing

### ISSUE THREAD

**Issue by hellojialee** [guohengkai/region-ensemble-network#3]
HI~
Thank you for your great work!
Could you please explain how do you get the hand center for me? 
I see you have given the hand center coordinate in   /labels/*_center.txt

Thank you very much!

**Comment by guohengkai** [comment]
Hi, we simply use the center of hand region, which is obtained by thresholding on the depth images. We will release the center computation scripts in the future.

**Comment by hellojialee** [comment]
@guohengkai 
Thank you for your replying!
 I have one more question: in your preprint paper, you haven't explain what loss function of the pose regression you adopt. Could you tell us?
Thank you~

**Comment by guohengkai** [comment]
Due to the limitation of paper length, we do not explain the loss. In fact, we use smooth L1 loss as that in faster R-CNN to reduce the effect of outliers in labels.

**Comment by hellojialee** [comment]
@guohengkai  
Did the loss accumulate all the considered 14 joints L1 loss equally?   Did you mean simply add the L1 norm difference between  all the predictions and ground truth ?

One more question: in the script compute_error.py 
    elif dataset == 'nyu':
        joint_idx = [0, 1, 2, 5, 3, 13, 12, 11, 10, 9, 8, 7, 6, 14];
        names = ['Palm', 'Wrist1', 'Wrist2', 'Thumb.R', 'Thumb.T', 'Index.R', 'Index.T', 'Mid.R', 'Mid.T', 'Ring.R', 'Ring.T', '...'Mean']
Did the joints evaluated in you result correspond the same 14 joint locations of other paper? Because the joint names in your code are not the same as NYU hand dataset.  I'm not sure if I explain my issue clearly.    
Sorry for bothering you again. And appreciate your kind reply.

**Comment by guohengkai** [comment]
1. We view each element in the vector (42 elements) equally.
2. They are the same as other papers. Sorry for the confusing names.

**Comment by hellojialee** [comment]
soga,    :)  Thank you for your selfless sharing!

**Comment by hellojialee** [comment]
HI, I come here again. 
In NYU hand dataset, there are two subsets consist of two different people. In other peoples' work, they resize the two subsets' hand region to different sizes, for example 300 pixel and 300*0.87 just as:https://github.com/xingyizhou/DeepModel/blob/master/training/GetH5DataNYU.py    line 64
So, did you considered how to deal with a new hand whose length is different from the NYU hand dataset?
Looking forward for your replying.
Thank you!

**Comment by guohengkai** [comment]
No, we did not. But when training the networks, we applied some augmentations including scaling, which can make it more robust to hand length variance.

**Comment by hellojialee** [comment]
But scaling the depth image means variance of joint coordinations at the same time?

**Comment by guohengkai** [comment]
The length of hand also effects the coordinations of joints, right?

**Comment by hellojialee** [comment]
Yes! We have to transform the coordination at the same time.

**Comment by guohengkai** [comment]
The center script has been released.
